WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -974,6 +1000,18 @@ - (void)loginValidationFailedWithError:(NSError *)error
     }
 }
 
+- (NSString *)getTagsCountPresentableString:(NSInteger)tagCount
+{
+    NSString *format = NSLocalizedString(@"%@ Tags", @"The number of tags in the writting settings. Plural. %@ is a placeholder for the number");
+    
+    if (tagCount == 1) {
```

Comment:
Just making a note here that this is the English-centric way of pluralizing, however, I don't think we're doing any better anywhere else in the code 😜